### ISSUE TITLE
Test fail on windows with build ccd43d6

### ISSUE THREAD

**Issue by lkppo** [metaeducation/ren-c#971]
[
        b2: copy lit ''[x y]
        b4: copy lit ''''[m n o p]
        m: make map! compose [
            a 0 'a 1 ''a 2 '''a 3 ''''a 4
            ((b2)) II ((b4)) IIII
        ]
        true
    ]
    Link to non-MANAGED item reached by GC
    C Source File ..\src\core\m-gc.c, Line 111, Pointer 0000000006562878
    At evaluator tick: 4636444
    Rebol Internal Error

---

The original test is:

    ; Currently, non-strict equality considers 'A and A to be equal, while strict
    ; equality consders them unequal.  Generalized quoting extends this to more
    ; quote levels...any number of quotes of the same value will be non-strict
    ; equal, while strict equality compares both properties.  This may not stick
    ; around as it merges case-insensitive comparison with type comparison.

    [
        (
            b2: copy lit ''[x y]
            b4: copy lit ''''[m n o p]
            m: make map! compose [
                a 0 'a 1 ''a 2 '''a 3 ''''a 4
                ((b2)) II ((b4)) IIII
            ]
            true
        )

        (0 = select/case m lit a)
        (1 = select/case m lit 'a)
        (2 = select/case m lit ''a)
        (3 = select/case m lit '''a)
        (4 = select/case m lit ''''a)

        ((trap [select m lit a])/id = 'conflicting-key)
        ((trap [m/(lit a)])/id = 'conflicting-key)

        ((trap [select m lit ''''a])/id = 'conflicting-key)
        ((trap [m/(lit ''''a)])/id = 'conflicting-key)

        ('II = m/[x y])
        ('IIII = m/[m n o p])

        ((trap [append b2 'z])/id = 'series-auto-locked)
        ((trap [append b4 'q])/id = 'series-auto-locked)
    ]

**Comment by lkppo** [comment]
Same error with test:

[
    unset 'a
    set lit '''''a <seta>
    <seta> = get lit ''a
]
Link to non-MANAGED item reached by GC
C Source File ..\src\core\m-gc.c, Line 111, Pointer 0000000007126F38
At evaluator tick: 6374748
Rebol Internal Error

**Comment by lkppo** [comment]
The failed test is:

;; Ren-C's QUOTED! is a generic and arbitrary-depth variant of the
;; LIT-XXX! types from historical Rebol.

;; SET and GET should see through escaping and work anyway

(
    unset 'a
    set lit '''''a <seta>
    <seta> = get lit ''a
)(
    unset 'a
    set lit 'a <seta>
    <seta> = get lit '''''''a
)(
    unset [a b]
    set ['''''a ''b] [<seta> <setb>]
    [<seta> <setb>] = get ['a '''''''b]
)

**Comment by hostilefork** [comment]
Thanks...this was the result of a bad rebase.  The bug doesn't exist in the commit I tested, which is after that one.

https://github.com/metaeducation/ren-c/commit/0959155022a12edc3049140a691c383d9824047d

I'm definitely in favor of getting the tests run somewhere on CI.  But I don't necessarily want that to be on Travis.  Shixin set it up on AppVeyor, but it broke and I didn't have access to fix it, so I just disabled it vs. getting notices of it breaking on every commit.  There's talk about Atronix coming back and getting involved again after their current deadline...they say they're coming to the conference in July.  So maybe there'll be more hands for these things...

*(Note: If you try to run the tests with an unoptimized build right now, you will be waiting a *long* time for the source analysis.  I'm just letting the source analysis slide for a little while.  After we have some more confidence in UTF-8 Everywhere, I'll kick in the ASCII optimization, which will make a huge difference there.)*
